core/region_test.go

```diff
@@ -291,3 +292,162 @@ func TestContainsString(t *testing.T) {
 		})
 	}
 }
+
+func TestFilterAsgs(t *testing.T) {
+	tests := []struct {
+		name    string
+		want    []string
+		tregion *region
```

Comment:
Is it typo? Or for `test region` or a copy/paste?

Reply:
Went with the consistency from the TestRequestSpotInstanceTypes(..) test.